eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -39,7 +39,27 @@ steps:
         echo "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT;]$(pwd)"
     # Execute only for Mac and if there is at least one dependency to be installed
     condition: and(succeeded(), contains(variables['OSVmImage'], 'macOS'), not(eq(variables['${{ parameters.DependenciesVariableName }}'], '')))
-    displayName: vcpkg bootstrap
+    displayName: vcpkg bootstrap (Mac)
+
+  # Update vcpkg and re-bootstrap on non-Mac OS'. Vcpkg is installed on the
+  # image at a given time stamp and not moved forward until some update
+  # procedure is run.
+  - task: Bash@3
+    inputs:
+      targetType: inline
+      script: |
+        cd $VCPKG_INSTALLATION_ROOT
+        git pull
+
+        ./bootstrap-vcpkg.sh
```

Comment:
for windows it is: `\bootstrap-vcpkg.bat`.   not sure if sh will be always compatible

Reply:
We're running the command in `bash` for both platforms so the command will succeed in both cases.